README.md

```diff
@@ -65,6 +69,22 @@ autotools configure command may need to be specified. This includes the
 and the ```--enable-venv``` option to build a Python venv containing the
 Ocean Model Grid Generator script and all python dependencies.
 
+## Contributing to NCTools
+
+The CNTools project encourages contributions from external parties.
+This includes from bug fixes and  enhancements to existing code, to adding
```

Comment:
"This includes from" wording is a little awkward.

Reply:
fixed